store/src/java/com/zimbra/cs/volume/VolumeCLI.java

```diff
@@ -393,22 +429,22 @@ private void validateEditCommand(VolumeInfo volumeInfo, Volume.StoreType volStor
             }
         } else if (volStoreType.equals(Volume.StoreType.EXTERNAL)) {
             if (!Strings.isNullOrEmpty(type)) {
-                throw new ParseException(A_TYPE + NOT_ALLOWED_EXTERNAL);
+                throw new ParseException(AdminConstants.A_TYPE + NOT_ALLOWED_EXTERNAL);
             }
             if (!Strings.isNullOrEmpty(path)) {
-                throw new ParseException(A_PATH + NOT_ALLOWED_EXTERNAL);
+                throw new ParseException(AdminConstants.A_PATH + NOT_ALLOWED_EXTERNAL);
             }
             if (!Strings.isNullOrEmpty(compress)) {
-                throw new ParseException(A_COMPRESS + NOT_ALLOWED_EXTERNAL);
+                throw new ParseException(AdminConstants.A_VOLUME_COMPRESS_BLOBS + NOT_ALLOWED_EXTERNAL);
```

Comment:
@dawoodshaikh will it be fine if the option name is changed?

Reply:
@RakeshAMore This is fine.

Reply:
Ok